### ISSUE TITLE
I tried to use example but there were some errors

### ISSUE THREAD

**Issue by skywalker512** [alosaur/angular_deno#3]
```deno run --config tsconfig.json --unstable --importmap=importmap.json --allow-read main.ts```
but show
```
error: Uncaught (in promise) Error: No platform exists!
      throw new Error("No platform exists!");
            ^
    at assertPlatform (https://jspm.dev/npm:@angular/core@11.0.7!cjs:35282:13)
    at https://jspm.dev/npm:@angular/core@11.0.7!cjs:35268:14
    at bootstrap (bootstrap.ts:31:67)
    at server.ts:17:24
```

**Comment by irustm** [comment]
Thanks!
Now I'm deliberately set Angular to 11.0.5

https://github.com/alosaur/angular_deno/pull/4

**Comment by irustm** [comment]
I will close this issue because it was fixed in #4. If you have more questions, please post them here
